tailwind.config.ts

```diff
@@ -11,7 +11,35 @@ const config = {
         '.storybook/**/*.{html,js,ts,tsx}',
     ],
     theme: {
-        extend: {},
+        extend: {
+            keyframes: {
+                'fade-in': {
```

Comment:
the new dialog doesn't play nice with the `tailwindcss-animate` lib, so I added custom animations that do the same effect. We probably will need our own animation definitions for the style guide anyway soon, so I figured we might as well start now